### PR TITLE
Add back Instagram links in nav bar

### PR DESCRIPTION
**Pull request by brandondean**
## Description
Not entirely sure why it was commented out, 51c5ebfdf0ab8b89cac7e81c9d082a61ed953e59 isn't really clear about that. This brings it back.

## Motivation and Context
#2345 

## How Has This Been Tested?
:eye: chromium 71, firefox 60.4, iOS safari
 
## Screenshots (if appropriate):
instagram-lists

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change API specifications or require data migrations)

### CODE REVIEW

**Comment by gregmarra** [comment]
LGTM, does it also work on mobile-width browsers?

**Comment by brandondean** [comment]
@gregmarra 
image from ios

**Comment by phil-lopreiato** [comment]
Yeah, we should probably bring twitter back too?

File: templates/base.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE html
```
Comment (andrewda) [inline templates/base.html]: ?

Comment (phil-lopreiato) [reply templates/base.html]: Whoops, should have caught that..

https://github.com/the-blue-alliance/the-blue-alliance/commit/7f0783285798349fd29ff42853b61ee4ed2fddc5

**Pull request merged**